device_plugins/nvidia_gpu/device-plugin-daemonset.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
Add a comment describing how this daemonset is expected to be used and what it will do.